libr/anal/sign.c

```diff
@@ -248,7 +248,7 @@ R_API bool r_sign_deserialize(RAnal *a, RSignItem *it, const char *k, const char
 	}
 
 	it->space = r_spaces_add (&a->zign_spaces, r_str_word_get0 (k2, 1));
-	it->name = r_str_new (r_str_word_get0 (k2, 2));
+	it->name = strdup (r_str_word_get0 (k2, 2));
```

Comment:
same here

Reply:
Yeah, that's why it is better to be careful when making changes like this. Previously when I was digging with it I messed up to such an extend that even `r2 -v` segfaulted XD